lang/cs/Org.Apache.REEF.Tests/Functional/IMRU/TestFailMapperEvaluatorsOnInit.cs

```diff
@@ -60,8 +60,11 @@ public override void TestFailedMapperOnLocalRuntime()
             var failedTaskCount = GetMessageCount(lines, FailedTaskMessage);
 
             // on each try each task should fail or complete or disappear with failed evaluator
-            // not all tasks will start successfully, so not checking this
+            // In each retry, there are 2 failed evaluators
+            // The running tasks should receive cancellation and return properly. There will be no failed task
             Assert.Equal((NumberOfRetry + 1) * numTasks, completedTaskCount + failedEvaluatorCount + failedTaskCount);
```

Comment:
We're asserting that `failedTaskCount = 0`, so we can remove `failedTaskCount` from this sum to simplify it